.github/workflows/ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: Tests
+
+on: pull_request
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
keep nightlies

Reply:
Am I being really stupid? As far as I can see, I should be able to use 3.10 alpha releases, but I don't see anything for nightlies. But, I can't get 3.10 to work either...

Reply:
Yeah, you could use alphas as nightlies. It's almost the same for our purposes.

If you want true nightlies, you'd need https://github.com/deadsnakes/action.